### ISSUE TITLE
How are the csv files created for training and testing?

### ISSUE THREAD

**Issue by lz75258** [mcx-mcx/EBAKER#1]
Thanks for your good work!  Can you please provide the ret3.csv file used for training and the rsicd_test.csv, rsitmd_test.csv and nwpu_test.csv files used for testing? Moreover, can you tell us how these files are constructed?

**Comment by mcx-mcx** [comment]
Hello, I have uploaded four CSV files in the main document for better reproduction. The composition of the test set is the same as the [GaLR](https://github.com/xiaoyuan1996/GaLR) work, and the training set has been filtered for duplicates across the three datasets using the p-hash method as per the RemoteCLIP approach like [this](https://github.com/ChenDelong1999/RemoteCLIP/issues/13).
